.ci/aarch64_linux/aarch64_wheel_ci_build.py

```diff
@@ -55,9 +55,22 @@ def build_ArmComputeLibrary() -> None:
         shutil.copytree(f"{acl_checkout_dir}/{d}", f"{acl_install_dir}/{d}")
 
 
-def update_wheel(wheel_path, desired_cuda) -> None:
+def replace_tag(filename) -> None:
```

Comment:
Is there not a better way to set the tag directly?

Reply:
Yes with auditwheel, however there are some issues with it. Will be looking next on converting everything to auditwheel repair.